controllers/booksController.js

```diff
@@ -0,0 +1,32 @@
+const Book = require('../model/Book');
+
+const handleNewBook = async(req, res) => {
+  console.log(req.body)
+  const { title, author } = req.body;
```

Comment:
Title is the only required property, as you will see in the schema. I added author here because I intend to include that as an optional field in the form anyways.